src/SunburstSettings.ts

```diff
@@ -88,21 +88,55 @@ class BaseFontCardSettings extends formattingSettings.FontControl {
 
 class SelectedCategoryGroup extends FormattingSettingsCard {
     public defaultShowSelected: boolean = true;
+    public defaultCustomizeStyle: boolean = false;
     public defaultFontSize: number = 14;
+    public defaultIndentation: number = 0;
 
     public showSelected = new formattingSettings.ToggleSwitch({
         name: "showSelected",
         displayNameKey: "Visual_ShowCategoryLabel",
         value: this.defaultShowSelected,
     });
-    public font = new BaseFontCardSettings(this.defaultFontSize);
+
+    public indentation = new formattingSettings.Slider({
+        name: "indentation",
+        displayNameKey: "Visual_Indentation",
+        value: this.defaultIndentation
+    });
+
+    public customizeStyle = new formattingSettings.ToggleSwitch({
+        name: "customizeStyle",
+        displayNameKey: "Visual_CustomizeStyle",
+        value: this.defaultCustomizeStyle,
+    });
+
+    public font = new BaseFontCardSettings(this.defaultFontSize, "categoryFont", "categoryFontFamily", "categoryFontSize", "categoryFontBold", "categoryFontItalic", "categoryFontUnderline");
```

Comment:
Let's rework this base class. We can simply pass unique keyword, such as "category", "label", etc and by default add to it all other strings ("category" + "Font" and same with others)